client/jsx/Header.jsx

```diff
@@ -11,6 +11,8 @@ Header = React.createClass({
     },
 
     logoClick() {
+        this.refs.searchFormInput.getDOMNode().value = '';
```

Comment:
https://facebook.github.io/react/docs/component-api.html#getdomnode - this function is deprecated. Plese, use uctual ReactDOM.findDOMNode. Thanks.